src/System.Private.ServiceModel/tools/test/BridgeCertificateBootstrapper/Bootstrapper.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Infrastructure.Common;
+
+public static class BridgeCertificateBootstrapper
+{
+    public static int Main(string[] args)
+    {
+        DisplayBanner(); 
+
+        if (args.Length < 1)
+        {
+            DisplayUsage();
+            return -1;
+        }
+
+        var filename = args[0];
+
+        byte[] certificateBytes; 
+        if (GetCertificate(out certificateBytes))
+        {
+            File.WriteAllBytes(filename, certificateBytes);
```

Comment:
Should probably try/catch around the WriteAllBytes and report failures.  One can foresee IO exceptions.